modules/oeFax/index.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Fax SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2018-2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace Modules\oeFax\Controller;
+
+require_once(__DIR__ . "/../../interface/globals.php");
+
+AppDispatch::getApiService();
```

Comment:
Doesn't this function need a `use Modules\oeFax\Controller\AppDispatch;` statement in script?